Sources/Dependencies/DependencyValues.swift

```diff
@@ -336,3 +336,9 @@ private let defaultContext: DependencyContext = {
 }()
 
 // TODO: should we have `@Dependency(\.runtimeWarningsEnabled)` and/or `@Dependency(\.treatWarningsAsErrors)`?
+
+#if swift(>=5.6)
+  public typealias _Sendable = Sendable
```

Comment:
Worth documenting in case people come across it?

```suggestion
  /// A typealias to Swift's `Sendable` protocol used to maintain backwards
  /// compatibility with Swift 5.5.
  public typealias _Sendable = Sendable
```